avalon/nuke/pipeline.py

```diff
@@ -239,17 +265,16 @@ def uninstall(config):
 
 
 def _install_menu():
+    """Installing Avalon menu to Nuke
+    """
     from ..tools import (
         creator,
-        # publish,
+        publish,
         workfiles,
-        loader,
-        sceneinventory,
+        cbloader,
```

Comment:
This will break things, as current master has them refactored to `loader` and `sceneinventory` already.